estudios_socioeconomicos/test_models.py

```diff
@@ -49,6 +49,15 @@ def test_estudio_str(self):
                                             status=self.estudio.status)
         self.assertEqual(str(self.estudio), expected)
 
+    def test_anwser_generation_for_study(self):
+        """ Tests whether the trigger that generates answers for each
+            new study works.
+        """
+        respuestas = Respuesta.objects.filter(estudio=self.estudio)
```

Comment:
Great! Here are you comparing the number of answers of the submitted study to the quantity it should have according to line 57?